server/migrations/20160212102110-add-avg-to-snippet.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
```

Comment:
@dwoznicka we should also add code which count `avg` for existing records in production